src/test/java/seedu/address/storage/JsonAdaptedConsultationTest.java

```diff
@@ -0,0 +1,102 @@
+package seedu.address.storage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.storage.JsonAdaptedConsultation.MISSING_FIELD_MESSAGE_FORMAT;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalConsults.BENSON_GROUP_CONSULT;
+import static seedu.address.testutil.TypicalPersons.BENSON;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.consultation.Address;
+import seedu.address.model.consultation.Consultation;
+import seedu.address.model.consultation.Day;
+import seedu.address.model.consultation.Time;
+import seedu.address.model.consultation.Type;
+import seedu.address.model.person.Name;
+import seedu.address.testutil.TypicalConsults;
+
+public class JsonAdaptedConsultationTest {
+    private static final String INVALID_NAME = "B@ns0n";
+    private static final String INVALID_DAY = "next Monday";
+    private static final String INVALID_TIME = "25:00";
+
+    private static final Consultation BENSON_CONSULT = BENSON_GROUP_CONSULT;
+    private static final String VALID_NAME = BENSON.getName().toString();
+    private static final String VALID_DAY = TypicalConsults.BENSON_DAY;
+    private static final String VALID_TIME = TypicalConsults.BENSON_TIME;
+    private static final String VALID_LOCATION = TypicalConsults.BENSON_LOCATION;
+    private static final String VALID_CONSULT_TYPE = TypicalConsults.BENSON_CONSULT_TYPE;
+
+
+    @Test
+    public void toModelType_validPersonDetails_returnsPerson() throws Exception {
```

Comment:
Rename the tests containing person to consultations/consults

Reply:
Nice catch! Thanks!

Reply:
Fixed in https://github.com/AY2021S1-CS2103T-F11-1/tp/pull/182/commits/37685c47c5ac4632eb43c995b8a213c4d0867dfc